src/styles/_catalog.scss

```diff
@@ -15,8 +15,9 @@
     border-radius: 5px;
     margin-bottom: 48px;
     transition-duration: 0.3s;
+    transition-property: color transform;
```

Comment:
I suggest that it didn’t work correctly. Open dev tools and check (I can’t because you didn’t update the build and your demo link without the last changes)

But on dev tools something like this 👇
<img width="298" alt="Screenshot 2022-07-22 at 20 24 16" src="https://user-images.githubusercontent.com/47606379/180492123-f4ce53a2-578a-426b-8756-f68b42842789.png">